charts/pega/templates/pega-environment-config.yaml

```diff
@@ -54,5 +54,14 @@ data:
   PEGA_STREAM_URL: {{ .Values.stream.url }}
 {{- end }}
 {{- end }}
-
+{{ if .Values.hazelcast.enabled }}
+  HZ_DISCOVERY_K8S: "true"
+  HZ_CLIENT_MODE: "true"
+{{ if .Values.hazelcast.client.clusterName }}
+  # Hazelcast Cluster Name
+  HZ_CLUSTER_NAME: {{ .Values.hazelcast.client.clusterName }}
+{{- end }}
+  # DNS of hazelcast service
+  HZ_SERVER_URL: {{ template "hazelcastName" . }}-service.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
This is not a URL it is a hostname.   I think this name is confusing. Comment should be updated as well.

Reply:
ok...I missed updating the comment. Changing the comment and environment variable name.